utils/vlog/vlog_to_pbtype.py

```diff
@@ -57,68 +57,6 @@
 
 from lib import xmlinc
 
-parser = argparse.ArgumentParser(
```

Comment:
What happened here?

Reply:
Previously bad merge it seems. https://github.com/SymbiFlow/symbiflow-arch-defs/pull/808/commits/9949ab4fbe5e2dea89a31f27566856c8bbeb51c9

https://github.com/SymbiFlow/symbiflow-arch-defs/blob/0ab6aaa1fd52db641996f8f15ae9f09bc25cd841/utils/vlog/vlog_to_pbtype.py#L60-L120

https://github.com/SymbiFlow/symbiflow-arch-defs/blob/0ab6aaa1fd52db641996f8f15ae9f09bc25cd841/utils/vlog/vlog_to_pbtype.py#L504-L570